src/software/simulated_tests/corner_kick_play_test.cpp

```diff
@@ -15,6 +15,7 @@ class CornerKickPlayTest : public SimulatedTestFixture
 
 TEST_F(CornerKickPlayTest, test_corner_kick_play)
 {
+    enableVisualizer();
```

Comment:
this should be removed for the final submission (CI won't pass)

Reply:
removed

Reply:
removed